resources/campaigns/Operation_Mole_Cricket_2010.json

```
@@ -0,0 +1,11 @@
+{
+  "name": "Syria - Operation Mole Cricket 2010",
+  "theater": "Syria",
+  "authors": "Mustang25",
+  "recommended_player_faction": "Bluefor Modern",
+  "recommended_enemy_faction": "Syria 2011",
+  "description": "<p>In a scenario reminescent of the First Lebanon War, hostile Syrian-backed forces have flooded into the Bekaa Valley.</p><p>The objective of this operation is twofold: drive the enemy out of the Bekaa Valley and push past the Golan Heights into Syrian territory to capture Tiyas Airbase.</p>",
+  "version": 6.0,

```

```suggestion
  "version": "6.0",
```

For future reference this does need to be in quotes. 6.10 == 6.1. "6.10" != "6.1"